pydicom/__init__.py

```diff
@@ -49,5 +48,4 @@
            'dcmwrite',
            'read_file',
            'write_file',
-           '__version__',
-           '__version_info__']
```

Comment:
I don't see __version__info replaced anywhere else -- maybe should keep in case anyone used it?  Can it just be generated from the version parsing?

Reply:
True I did not think about it. I will fix it.